### ISSUE TITLE
csv import from github doesn't work

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/dlme#114]
```
Octokit::Forbidden: GET https://api.github.com/repos/waynegraham/dlme-metadata/contents
/museum/records/met_museum_records.csv: 403 - This API returns blobs up to 1 MB in size. The
 requested blob is too...
```

**Comment by jcoyne** [comment]
@cbeer fixed by #116 ?

**Comment by cbeer** [comment]
Yes.
